vcd/resource_vcd_snat.go

```diff
@@ -69,13 +76,16 @@ func resourceVcdSNATCreate(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf(errorUnableToFindEdgeGateway, err)
 	}
 
+	// TODO add support of external network
 	var orgVdcnetwork *types.OrgVDCNetwork
 	providedNetworkName := d.Get("network_name")
 	if nil != providedNetworkName && "" != providedNetworkName.(string) {
 		orgVdcnetwork, err = getNetwork(d, vcdClient, providedNetworkName.(string))
-	} else {
-		_, _ = fmt.Fprint(GetTerraformStdout(), "WARNING: this resource will require network_name in the next major version \n")
 	}
+	// TODO enable when external network supported
+	/*else {
+		_, _ = fmt.Fprint(GetTerraformStdout(), "WARNING: this resource will require network_name in the next major version \n")
```

Comment:
Please add `network_type and network_name` to the message so we don't forget later.

Reply:
Changed